internal/handlers/pg/pgdb/pool.go

```diff
@@ -203,11 +205,53 @@ func (pgPool *Pool) Schemas(ctx context.Context) ([]string, error) {
 	return res, nil
 }
 
+// Collections returns a sorted list of FerretDB collection names.
+func (pgPool *Pool) Collections(ctx context.Context, schema string) ([]string, error) {
+	tx, err := pgPool.Begin(ctx)
+	if err != nil {
+		return nil, lazyerrors.Error(err)
+	}
+	defer tx.Rollback(ctx)
```

Comment:
So, we always want to rollback? Why?

Reply:
We don't want to change anything here but load settings table. `getSettingsTable` accepts transaction, so I have to create one. I can probably change `Rollback` to `Commit` but `Rollback` looked more logically correct to me.

Reply:
> We don't want to change anything here but load settings table. `getSettingsTable` accepts transaction, so I have to create one. I can probably change `Rollback` to `Commit` but `Rollback` looked more logically correct to me.

Let's add a comment here right as you wrote in the comment explanation.

Reply:
+1! With that explanation, rollback makes sense (that was also my guess), but let's document it for clarity.

Reply:
Added.

Reply:
> Added.

I expected you to add exactly as in the comment:
> We don't want to change anything here but load settings table. getSettingsTable accepts transaction, so I have to create one.

^^ and this information are not obvious.

But you've added:
> call Rollback() here as we don't change anything in the database

but that's obvious to do a rollback if you don't want change!

Reply:
Added comment to transaction creation.